src/Model/Package.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use BringYourOwnIdeas\Maintenance\Tasks\UpdatePackageInfo;
+
+/**
+ * Describes an installed composer package version.
+ */
+class Package extends DataObject
+{
+    private static $db = [
+        'Name' => 'Varchar(255)',
+        'Description' => 'Varchar(255)',
+        'Version' => 'Varchar(255)',
+        'Type' => 'Varchar(255)',
+    ];
+
+    private static $summary_fields = [
+        'Title',
+        'Description',
+        'Version',
+    ];
+
+    /**
+     * Strips vendor and 'silverstripe-' prefix from Name property
+     * @return string More easily digestable module name for human consumers
+     */
+    public function getTitle()
+    {
+        return preg_replace('#^[^/]+/(silverstripe-)?#', '', $this->Name);
+    }
```

Comment:
This definitely needs a unit test with a bunch of different inputs, but can be done after merge

Reply:
done.